docs/service-mesh/10-03-connection-refused.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Connection refused errors after upgrade
+title: Connection refused errors
 type: Troubleshooting
 ---
 
-Mutual TLS (mTLS) is enabled in the Service Mesh by default. As a result, every element of the Service Mesh must have an Istio sidecar to allow TLS communication. If you don't want to use a sidecar, you can whitelist a service and disable TLS traffic for it by creating a [DestinationRule](https://istio.io/docs/reference/config/networking/destination-rule/).  
+Mutual TLS (mTLS) is enabled in the Service Mesh by default. As a result, every element of the Service Mesh must have an Istio sidecar with a valid TLS certificate to allow communication. Connection between a service with a sidecar and a service without a sidecar results in a `Connection reset by peer` or a `GOAWAY` response. 
 
 - To enable sidecar injection for Pods of existing Services, restart them after upgrading to Kyma 1.0 or higher.
-- To create a DestinationRule for a Service, follow the official [Istio documentation](https://istio.io/docs/reference/config/networking/destination-rule/).
+- If you don't want to use a sidecar, you can whitelist a Service and disable TLS traffic for it by creating a [DestinationRule](https://istio.io/docs/reference/config/networking/destination-rule/).  
```

Comment:
To tell all the clients(other sidecars) to do not use mTLS when connecting to my service we use DestinationRule. To accept PERMISSIVE connections for my service that has a sidecar we use Policy - at least for now as there are now in 1.5 new CRs for that, but this is out of scope as we don't have it yet. To disable sidecar injection we follow step bellow :)